plutus-playground-server/app/Main.hs

```diff
@@ -55,10 +56,16 @@ webserverCommandParser =
                 (short 'p' <> long "port" <> help "Webserver port number" <>
                  showDefault <>
                  value 8080)
+        _maxInterpretationTime <-
+            option
+                auto
+                (short 'i' <> long "interpretation" <> help "Max interpretation time" <>
```

Comment:
the help text should say the unit (seconds)

Reply:
Changed to  "Max interpretation time (seconds)"